.rubocop_todo.yml

```diff
@@ -38,53 +49,63 @@ Lint/UnusedBlockArgument:
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Offense count: 508
+# Offense count: 3
+# Configuration parameters: ContextCreatingMethods.
+Lint/UselessAccessModifier:
+  Enabled: false
+
+# Offense count: 524
 Metrics/AbcSize:
   Max: 114
 
-# Offense count: 13
+# Offense count: 12
 Metrics/BlockNesting:
   Max: 5
 
-# Offense count: 49
+# Offense count: 50
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 800
+  Max: 780
 
-# Offense count: 134
+# Offense count: 131
 Metrics/CyclomaticComplexity:
   Max: 24
 
-# Offense count: 8167
+# Offense count: 8762
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes.
 # URISchemes: http, https
 Metrics/LineLength:
   Max: 385
 
-# Offense count: 390
+# Offense count: 433
 # Configuration parameters: CountComments.
 Metrics/MethodLength:
   Max: 94
 
 # Offense count: 21
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 450
+  Max: 416
 
-# Offense count: 5
+# Offense count: 9
 # Configuration parameters: CountKeywordArgs.
 Metrics/ParameterLists:
   Max: 7
 
-# Offense count: 105
+# Offense count: 98
 Metrics/PerceivedComplexity:
   Max: 24
 
-# Offense count: 5
+# Offense count: 6
 # Cop supports --auto-correct.
 Performance/Casecmp:
   Enabled: false
 
+# Offense count: 17
+# Cop supports --auto-correct.
+Performance/PushSplat:
+  Enabled: false
```

Comment:
I understand why this rule makes sense - if you're combining two arrays, why splat one and push it to the other when you can just use the concat method - but I agree the documentation makes no sense